discovery/spec.md

```diff
@@ -365,775 +244,693 @@ The following sections define the attributes that appear in a Service entity.
     }
     ````
 
-##### specversions
+#### channel
 
-- Type: Array of `String` values
-- Description: CloudEvents
-  [`specversions`](../cloudevents/spec.md#specversion)
-  that can be used for events published for this service.
+- Type: String
+- Description: A string that can be used to correlate Endpoints. Any Endpoints
+  within an instance of a Discovery Service that share the same `channel` value
+  MUST have some relationship. This specification does not define that
+  relationship or the specific values used in this property. 
+
+  This property MAY be serialized as an empty string or MAY excluded from the
+  serialization entirely if there is no value.
 - Constraints:
-  - REQUIRED
-  - MUST be a non-empty array of non-empty strings
+  - OPTIONAL
+  - if present, MUST be a string
+- Examples:
+  - `queue1`
+
+#### authscope
+
+- Type: TBD
+- Description: TBD
+- Constraints:
+  - OPTIONAL
+- Examples:
+  - TBD
 
-##### subscriptionurl
+#### usage
```

Comment:
`usage` seems odd to me. I would call this `role`.

Reply:
Let's ask on the call... I think `role` did pop up in our off-line chats but I can't remember why we didn't use it. It may have been because we wanted a word that implied it was the role as viewed by the "user of the endpoint" instead of it's "role as viewed from the endpoint's perspective".